templates/fsharp/xplat/AvaloniaTest.Web/AvaloniaTest.Web.fsproj

```diff
@@ -1,42 +1,31 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think this file should look exactly like AvaloniaTest.Web.csproj,please update

Reply:
They now look the same except F# includes these files because unlike C# they are not inlcuded by default in F# projects. Some of them are required to be inlcuded (Program.fs, runtimeconfig.template.json), others are included to they are shown in VS. For the blazor project I guess the Blazor SDK took care of ensuring wwwroot and launchsettings are shown, but without these lines AppBundles folder and launchsetttings would not be shown in VS

Reply:
I meant removing <EmccTotalMemory>16777216</EmccTotalMemory> thingy,thanks ;)